polylogue/cli/init.py

```
@@ -46,20 +47,33 @@ def run_init_cli(args: argparse.Namespace, env: CommandEnv) -> None:
     console.print("\n[bold cyan]Welcome to Polylogue![/bold cyan]")
     console.print("Let's set up your configuration.\n")
 
-    # Step 1: Output directory
-    default_dir = Path.home() / "polylogue-data"
+    # Step 1: Roots
+    default_output_root = CONFIG.defaults.output_dirs.render.parent
+    default_input_root = CONFIG.exports.chatgpt
     console.print(f"[bold]Output Directory[/bold]")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)